### PR TITLE
#752: changed save/cancel buttons in RiskLog

### PR DESCRIPTION
**Pull request by vonbuelow**
## Changes

Changed the save/cancel buttons in the RiskLog component to use the NERSuccessButton/NERFailButton component rather than just the plain MUI button

## Test Cases

- yarn test:frontend 

## Screenshots

Screenshot_20230206_104319

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (752)

### CODE REVIEW

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx [hunk 1]
```diff
@@ -215,8 +217,12 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
             />
           </DialogContent>
           <DialogActions>
-            <Button onClick={handleClose}>Close</Button>
-            <Button type="submit">Save Changes</Button>
+            <NERFailButton onClick={handleClose} sx={{ mx: 2 }}>
+              Closesx=
```
Comment (tigerhippo) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:221]: Is there a specific reason why you said 'Closesx=' and not 'Closes'?

**Comment by tigerhippo** [comment]
@RChandler234 please also review this, I could have missed stuff

**Comment by tigerhippo** [comment]
Looks good + ready to merge!

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx [hunk 2]
```diff
@@ -215,8 +217,12 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
             />
           </DialogContent>
           <DialogActions>
-            <Button onClick={handleClose}>Close</Button>
-            <Button type="submit">Save Changes</Button>
+            <NERFailButton onClick={handleClose} sx={{ mx: 1 }}>
+              Closes
```
Comment (RChandler234) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:221]: Should say 'Close'

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by anthonybernardi — APPROVED** [review]
awesome!

**Pull request merged**